python/Ganga/GPIDev/Lib/Registry/PrepRegistry.py

```diff
@@ -14,14 +14,9 @@
 
 class PrepRegistry(Registry):
 
-    def __init__(self, name, doc, dirty_flush_counter=10, update_index_time=30, dirty_max_timeout=60, dirty_min_timeout=30):
-
-        super(PrepRegistry, self).__init__(name, doc, dirty_flush_counter, update_index_time, dirty_max_timeout, dirty_min_timeout)
-
-        self.releaseThread = threading.Thread(target=self.trackandRelease, args=())
-        self.releaseThread.daemon = True
-        self.releaseThread.start()
```

Comment:
This causes us to end up back with problems with multiple sessions trying to submit jobs.
I'd rather keep the track and release code as there will be complaints from some users if this goes